db/db_iter.cc

```diff
@@ -968,6 +997,10 @@ bool DBIter::FindValueForCurrentKeyUsingSeek() {
             ikey, RangeDelPositioningMode::kForwardTraversal)) {
       break;
     } else if (ikey.type == kTypeValue) {
```

Comment:
Can we make it an "else" here and consolidate the two `iter_.PrepareValue()` calls?